scripts/HealthCheck.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.digital.ho.hocs;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+/**
+ * Class used for container health check by docker compose
+ * Pass address to be health checked as the single command line argument
+ */
+public class HealthCheck {
+
+    public static void main(String[] args) throws IOException, InterruptedException {
+
+        if(args.length < 1){
```

Comment:
We can get rid of the need for this by using the same mechanic as here - https://github.com/UKHomeOffice/hocs-casework/blob/eaf75d3fc0de95bdd864ffc4914df1cb4c71b586/scripts/CsvCaseDataUpdater.java#L35

Would allow for this to handle itself internally based on the server port we specify for hocs-notify to run on 🙂

Reply:
Probably. However, this code is standard across all services atm so I'd make that change under another ticket (across all services) instead of introducing a discrepancy. Personally I'd rather use curl.